tests/metagen/typegraphs/identities/py/fdk.py

```
@@ -0,0 +1,1675 @@
+    def PyPrimitives():
+        return_node = NodeDescs.Primitives()
+        return NodeMeta(
+            sub_nodes=return_node.sub_nodes,
+            variants=return_node.variants,
+            arg_types={
+                "data": "primitives",
+            },
+        )
+
+PrimitivesArgs = typing.TypedDict("PrimitivesArgs", {
+    "data": Primitives,
```

Ruff (0.8.2)

Undefined name `Primitives`

(F821)